lib/Cron/Workers/SimpleWorker.php

```diff
@@ -150,21 +150,20 @@ function reschedule(ScheduledTask $task, $timeout) {
   }
 
   private function isInProgress(ScheduledTask $task) {
-    $meta = $task->getMeta();
-    if (!empty($meta['in_progress'])) {
+    if (!empty($task->in_progress)) {
```

Comment:
This could be refactored just to `return !empty($task->in_progress)`, but it doesn't have to be addressed.